opm/grid/MinpvProcessor.hpp

```diff
@@ -205,7 +218,7 @@ namespace Opm
 
                             // Add a connection if the cell above and below is active and has porv > minpv
                             if ((actnum.empty() || (actnum[c_above] && actnum[c_below])) && pv[c_above] > minpvv[c_above] && pv[c_below] > minpvv[c_below]) {
-                                    nnc.insert(std::make_pair(c_above, c_below));
+                                    nnc.emplace_back(c_above, c_below, c);
```

Comment:
The original code is complicated and it has been a long time since I read it carefully., hence I might be wrong:
My interpretation was that this algorithm might remove more than one active cell for each nnc added. But the downstream PR seems to assume that this cannot happen

Reply:
> The original code is complicated 

You can say that!

> My interpretation was that this algorithm might remove more than one active cell for each nnc added

That is right - in the current code that will be multiple `PinchPair`objects with the same `cell1` and `cell2` value and different values for `removed_cell`. Eventually the `cell1, cell2` pairs are stored in a `std::set<>` which should handle the duplicates gracefully.

Reply:
Well maybe I am wrong. What if the top-most cell (k==0) is pinched and hence no nnc created?